Library/Homebrew/global.rb

```diff
@@ -59,7 +59,10 @@
 module Homebrew
   extend FileUtils
 
-  DEFAULT_PREFIX ||= "/usr/local"
+  HOMEBREW_DEFAULT_PREFIX = "/home/linuxbrew/.linuxbrew"
```

Comment:
Wrong way around. Also, use this variable in `Library/Homebrew/os/linux/global.rb` too.

Reply:
Oh snap, you're right, my bad.

I get this if I use those constants in `Library/Homebrew/os/linux/global.rb`:
```
/home/dawidd6/github/dawidd6/brew/Library/Homebrew/os/linux/global.rb:7:in `<module:Homebrew>': uninitialized constant Homebrew::LINUXBREW_DEFAULT_PREFIX (NameError)
	from /home/dawidd6/github/dawidd6/brew/Library/Homebrew/os/linux/global.rb:3:in `<top (required)>'
	from /home/dawidd6/github/dawidd6/brew/Library/Homebrew/os/global.rb:3:in `require'
	from /home/dawidd6/github/dawidd6/brew/Library/Homebrew/os/global.rb:3:in `<top (required)>'
	from /home/dawidd6/github/dawidd6/brew/Library/Homebrew/global.rb:57:in `require'
	from /home/dawidd6/github/dawidd6/brew/Library/Homebrew/global.rb:57:in `<top (required)>'
	from /home/dawidd6/github/dawidd6/brew/Library/Homebrew/brew.rb:23:in `require_relative'
	from /home/dawidd6/github/dawidd6/brew/Library/Homebrew/brew.rb:23:in `<main>'
```

Reply:
You may need to update the namespace to ensure it's fine. It's definitely possible as it's included after.